patroni/dcs/etcd.py

```diff
@@ -215,6 +215,13 @@ def get_etcd_client(config):
                 sleep(5)
         return client
 
+    def set_ttl(self, ttl):
+        ttl = int(ttl)
+        if self._ttl != ttl:
+            self.reset_cluster()
```

Comment:
This is exactly the kind of explanation that should become the comment to this method.